docs/developer-manual/terms-of-service.md

```diff
@@ -0,0 +1,157 @@
+
+## THE SERVICES
+Collectively, the Oracle data collected, calculated, validated and published
+by Orcfax to transactions on the Cardano blockchain; along with the related Archival 
+data and files that Orcfax archives to the Arweave network; as well as the data and
+information published to the Website (including any and all sub-domain sites such as
+the Orcfax Explorer and Docs sites) are hereby referred to, collectively, as the 
+"Services" provided by Orcfax. 
+
+## REVISION OF TERMS OF SERVICE
+We reserve the right, in our sole discretion, to modify these Terms from time to
+time. If we make changes, we will provide you with notice of such changes using
+Commercially Reasonable Means, which is defined as taking the desired notification
+action without a material increase in costs incurred by Orcfax and the competitive
+environment in which such action occurs. Examples of such notice include providing
+a notice through the Website or updating the date at the top of these Terms. Unless
+we say otherwise in our notice, any modifications are effective immediately, and your
+continued use of Orcfax's Services will confirm your acceptance of the changes. 
+If you do not agree to the amended Terms, you must stop using our Services.
+
+## YOUR AGREEMENT TO THESE TERMS OF SERVICE
+These Terms describe your rights and obligations in relation to the Services and
+our disclaimers and limitations of legal liability for the Services. By accessing
+or using any part of Orcfax's Oracle data, Archival data or Website data you agree to
+become bound by these Terms of Service.
+
+If you use the Services on behalf of a company, organization (including a
+decentralized autonomous organization or “DAO”) or other entity, or if you
+are an automated agent acting on behalf of another entity (including "bots" that
+execute actions using artificial intellegence or other technologies for autonomous
+decision-making, such as "smart contracts"), then “you” includes you and that entity. 
+You represent and warrant that (a) you are an authorized representative of the
+entity with the authority and agency to bind the entity to these Terms, and (b) you
+agree to these Terms on the entity’s behalf.
+
+You must not use any our Services, if you do not agree to these Terms or if you
+do not have the authority or agency to bind the entity, on whose behalf you are using
+the Services, to these Terms. 
+
+## YOUR REPRESENTATIONS AND WARRANTIES; CONDITIONS
+Our Services enable access to data from multiple and varied sources. You
+acknowledge and agree that the accuracy, availability and quality of data collected
+and calculated via our Services may be impacted by various factors, including as
+a result of the underlying data being volatile or otherwise compromised at the
+data source, which could include malfeance or malpractice.
+
+Some of our Services require holding and paying Distributed Ledger Technology (DLT) 
```

Comment:
We use "distributed ledger technology" in lines 92 and 148 do we want too use "DLT" there?

Reply:
we should use Distributed Ledger Technology (DLT) in all cases where we mean blockchain. We can then use the acronynm DLT if in same paragraph but if re-using the term again in a later paragraph I'd like to spell it out in full again.